content/400-pulse/200-getting-started.mdx

```diff
@@ -62,46 +81,91 @@ const prisma = new PrismaClient().$extends(
 )
 ```
 
-<Admonition type="note">
-
-You stored this API key in your .env file after [enabling Pulse](#1-enable-pulse). If needed, you can navigate to your respective project environment and generate a new API key.
 
-</Admonition>
 
 ### 2.3. Create your first Pulse subscription
 
 With the Pulse extension applied, you can use Pulse's `subscribe()` method on any model defined in your Prisma Schema to subscribe to data change events.
 
-In the below example, a subscription is made to a `notification` model that listens for _any_ change event on that table:
+In the below example, it is assumed that your Prisma schema has a `User` model. A subscription is made to that `User` model that listens for _any_ change event on that table:
+
+<TabbedContent code>
+<TabItem value="Subscription">
 
 ```ts
-const prisma = new PrismaClient().$extends(withPulse({ apiKey: apiKey }))
+import { PrismaClient } from '@prisma/client'
+import { withPulse } from '@prisma/extension-pulse'
```

Comment:
Can we mark the added/changed lines with color as in git diffs?

Reply:
yup, added the annotation to the code block to highlight the subscription lines: `highlight=9-13;add`